server/store/store.go

```diff
@@ -211,23 +256,21 @@ func (s *SQLStore) GetPostInfoByMattermostID(postID string) (*storemodels.PostIn
 
 func (s *SQLStore) LinkPosts(postInfo storemodels.PostInfo) error {
 	if s.driverName == "postgres" {
-		_, err := s.getQueryBuilder().Insert("msteamssync_posts").Columns("mmPostID, msTeamsPostID, msTeamsChannelID, msTeamsLastUpdateAt").Values(
+		if _, err := s.getQueryBuilder().Insert("msteamssync_posts").Columns("mmPostID, msTeamsPostID, msTeamsChannelID, msTeamsLastUpdateAt").Values(
```

Comment:
Just to undrestand this change. This has nothing to do with the MySQL support, right? it is just an improvement in the code style, right?